src/Tests/PathautoTokenTest.php

```diff
@@ -42,7 +42,7 @@ public function testPathautoTokens() {
     /* @var \Drupal\pathauto\PathautoManagerInterface $manager */
     $manager = \Drupal::service('pathauto.manager');
     $manager->cleanTokenValues($replacements, $data, array());
-    $this->assertEqual($replacements['[array:join-path]'], 'test-first-arg/array-value');
+    $this->assertEqual($replacements['[array:join]'], 'test-first-arg/array-value');
```

Comment:
That's because `join-path` is a special case of `join` that uses au `/` as separator (while `join` uses a `-` by default). We might want to test the `join` token too but this test was for the `join-path` token and shouldn't be changed.